styleguide/source/_patterns/01-atoms/headings.twig

```diff
@@ -1,3 +1,41 @@
+{%
+  set headerLevels = [
+    {
+      'level': '1',
+      'class': 'ama__h1'
+    },
+    {
+      'level': '2',
+      'class': 'ama__h2'
+    },
+    {
+      'level': '3',
+      'class': 'ama__h3'
+    },
+    {
+      'level': '4',
+      'class': 'ama__h4'
+    },
+    {
+      'level': '5',
+      'class': 'ama__h5'
+    },
+    {
+      'level': '6',
+      'class': 'ama__h6'
+    }
+  ]
+%}
+
 {% for heading in headings %}
+  {%
+    set heading = heading|merge(
+      {
+        level: headerLevels[loop.index0].level,
+        class: headerLevels[loop.index0].class
+      }
+    )
+  %}
```

Comment:
This is awesome!

Reply:
@nstriedinger actually helped out with this. It was questionable wether or not we should use this looping technique or just simply have h1, h2, h3 twig files

Reply:
Very cool, great collaboration!